bundles/thunk_bundle.js

```diff
@@ -5824,6 +5824,26 @@ module.exports = (() => {
 
   // schema.ts
   var import_pascalcase = __toESM(require_pascalcase());
+  function throwOnDynamicSchemaWithJsAutocompleteFunction(dynamicSchema, parentKey) {
+    if (!dynamicSchema) {
+      return;
+    }
+    if (Array.isArray(dynamicSchema)) {
+      dynamicSchema.forEach((item) => throwOnDynamicSchemaWithJsAutocompleteFunction(item));
+      return;
+    }
+    if (typeof dynamicSchema === "object") {
+      for (const key of Object.keys(dynamicSchema)) {
+        throwOnDynamicSchemaWithJsAutocompleteFunction(dynamicSchema[key], key);
+      }
+    }
+    if (typeof dynamicSchema === "function" && parentKey === "autocomplete") {
+      throw new UserVisibleError(
```

Comment:
I'd advise against a user-visible error here, since the Pack user can't do anything to remedy this mistake. I think it's better for the user to get a generic error message, and they will reach out to the Pack maker, who if they get access to the doc can see this error in the logs.

Reply:
Changed to `throw Error()` so a pack maker can still see a meaningful error but a pack user will just see that the pack had an internal error